config/options.go

```diff
@@ -1,5 +1,7 @@
 package config
 
+const MAX_FILE_SIZE_KB = 2 * 1024 // 2048 KiB (2 MiB)
```

Comment:
You can just put the value (without the constant variable) in the option.
Then we don't need a comment, but can mention the default value in the help message.
This also let's people looking at the help message and not the code know what the default is.

Reply:
Unresolving.

The option was updated, but this const was not removed.